tools/win32/detect_cpu_extensions_wine.py

```diff
@@ -68,14 +69,12 @@ def path_as_windows(fpath):
 
 def write_summary(allcodes):
     """Write a summary of all found codes to stdout."""
-    print """\n
-----------------------------------------------------------------------------
-Checked all binary files for CPU extension codes. Found the following codes:"""
+    print("\n"
+          "----------------------------------------------------------------------------"
+          "Checked all binary files for CPU extension codes. Found the following codes:")
```

Comment:
I'll make the change anyway. If the file's not used it can be removed in a future commit.